src/WebApi/Application/UseCases/QuestionList/GetQuestionList/IGetQuestionListUseCase.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+
+namespace Application.UseCases.QuestionList.GetQuestionList
+{
+    public interface IGetQuestionListUseCase
+    {
+        Task Execute(GetQuestionListInput input);
+        void SetOutputPort(IOutputPort outputPort);
+    }
```

Comment:
bruh, I've noticed you sometimes put empty lines between interface members, and here you for example don't.

Might be a non-specified standard so feel free to be the pioneer who picks one, but let's follow that standard. ☺️ 

Also I'm fairly sure we use file scoped namespaces here:
```cs
namespace Application.UseCases.QuestionList.GetQuestionList;

// ...
```